packages/react-admin/src/mui/list/Filter.js

```diff
@@ -23,20 +24,21 @@ export class Filter extends Component {
         this.filters = nextProps.filterValues;
     }
 
-    componentWillUnmount() {
-        if (this.props.setFilters) {
-            this.setFilters.cancel();
```

Comment:
this shouldn't be removed

Reply:
it should, because setFilters is not debounced anymore, it's moved down